tests/conftest.py

```diff
@@ -74,17 +75,30 @@ def pytest_collection_modifyitems(session, config, items):
 
     """
     for item in items[:]:
-        skip_marker = item.get_closest_marker("skipif_ocs_version")
-        if skip_marker:
-            skip_condition = skip_marker.args
-            log.info(skip_condition)
-            # skip_confition will be a tuple
-            # and condition will be first element in the tuple
-            if skipif_ocs_version(skip_condition[0]):
-                log.info(
-                    f'Test: {item} will be skipped due to {skip_condition}'
-                )
-                items.remove(item)
+        skip_marker = [
+            item.get_closest_marker("skipif_ocs_version"),
+            item.get_closest_marker("skipif_lso_deployment")
+        ]
+        for marker in skip_marker:
+            if marker:
+                if marker.name == 'skipif_ocs_version':
+                    skip_condition = skip_marker[0].args
+                    log.info(skip_condition)
+                    # skip_confition will be a tuple
+                    # and condition will be first element in the tuple
+                    if skipif_ocs_version(skip_condition[0]):
+                        log.info(
+                            f'Test: {item} will be skipped due to {skip_condition}'
+                        )
+                    items.remove(item)
+                    break
+                elif marker.name == 'skipif_lso_deployment':
+                    if skipif_lso_deployment():
```

Comment:
would be good to log the skip condition as done for skip_ocs_version

Reply:
There is no skip condition for skipif_lso_deployment.The function checks whether the deployment is LSO based or not and returns boolean based on it.